integration-tests/tests/ubuntuFan_test.go

```diff
@@ -1,5 +1,5 @@
 // -*- Mode: Go; indent-tabs-mode: t -*-
-// +build !excludeintegration
+// +build !excludeintegration,!excludeflaky
```

Comment:
@niemeyer thanks a lot for your comments :)

This concrete suite is only flaky when executed on low performance systems, otherwise it gives good and consistent results. I've updated the tag to make this clear, now it's excludeFlakyOnLowPerformance. The way it's defined makes that it will be executed by default and you have to explicitly exclude it, so that we won't ignore it in the long term.

In my opinion we should keep the suite, beside it checks the Ubuntu Fan feature it exercises some aspects of docker that are not tried elsewhere, especifically changing the default bridge and pulling images.